src/main/resources/datastore.yaml

```diff
@@ -0,0 +1,9 @@
+#
+# Datastore Server Configuration
```

Comment:
Point of clarity. This configuration is actually the `ByteBlobServicePod` configuration object right? I would love to rename this in the future (not right now) if so

Reply:
Yes it is. Agree that datastore.yaml is a bad name.